content/docs/user-guide/deploying/index.md

```diff
@@ -57,24 +57,24 @@ This will stop the deployment and erase deployment state value
 
 ## Making requests
 
-You also can create MLEM Client for your deployment to make some requests:
+You also can create a MLEM client for your deployment from Python code
```

Comment:
I linked to this section from the Docker server deployment section of the Get Started (which is the main/happy path). However the GS mentions `curl` and `mlem apply-remote`, which are not covered here.

- [ ] should this section of the guide be more comprehensive?

Reply:
> However the GS mentions curl and mlem apply-remote, which are not covered here

mentioning and giving a link to `doc/user-guide/serving/index.md` should be enough I think

Reply:
OK I see. The structure of the UG is a bit confusing then, bc/ we would link from a section about deployment to a guide about serving when there's a specific guide about deployment... ⏳

Reply:
I updated the link, but the guide and/or terminology is def. still quite confusing.

- [ ] We should clarify somehow (separate task)